src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10334,21 +10334,11 @@ public sealed partial class FrameworkName : System.IEquatable<System.Runtime.Ver
         public static bool operator !=(System.Runtime.Versioning.FrameworkName? left, System.Runtime.Versioning.FrameworkName? right) { throw null; }
         public override string ToString() { throw null; }
     }
-    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly | System.AttributeTargets.Class | System.AttributeTargets.Constructor | System.AttributeTargets.Enum | System.AttributeTargets.Event | System.AttributeTargets.Field | System.AttributeTargets.Interface | System.AttributeTargets.Method | System.AttributeTargets.Module | System.AttributeTargets.Property | System.AttributeTargets.Struct, AllowMultiple = true, Inherited = false)]
-    public sealed class SupportedOSPlatformAttribute : System.Runtime.Versioning.OSPlatformAttribute
```

Comment:
Seems because of the renaming from `MinimumOSPlatformAttribute`  => `SupportedOSPlatformAttribute` broke the alphabetical order, moved the API down, Same for  `RemovedOSPlatformAttribute`  => `UnsupportedOSPlatformAttribute`